contracts/oracle/YUnsafeOracleV1.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@lbertenasco/contract-utils/contracts/abstract/UtilsReady.sol";
+
+import "../interfaces/oracle/IYOracle.sol";
+import "../interfaces/oracle/ISimpleOracle.sol";
+
+contract YUnsafeOracleV1 is UtilsReady, IYOracle {
```

Comment:
Why is it called unsafe ?

Reply:
because is fairly easy to manipulate, and should only be used for non-value related functions.